### PR TITLE
transfer ict.senecacollege.ca img to current repo

### LINKED ISSUE #113: Bring all images from ict.senecacollege.ca into the repo
See https://github.com/Seneca-ICTOER/IPC144/search?q=https%3A%2F%2Fict.senecacollege.ca%2F for instances where we're still linking to (images) on the old IPC144 site.  These should all be changed to bring the images into this repo and linked to locally.

### LINKED ISSUE #126: Use /img vs. static/img for all image URLs
We have a [few places](https://github.com/Seneca-ICTOER/IPC144/search?q=static%2Fimg) where we include the `static/` dir in our image URLs.  We don't need this.  They can all be `/img/*`.

@Kevan-Y do you have interest in this, since you're working on the other image import?

### PR DESCRIPTION
**Pull request by Kevan-Y**
Fixes #126 
Fixes #113 

Check all pages, doesn't seem to have any issue.

### CODE REVIEW

File: docs/F-Refinements/character-strings.md [hunk 1]
```diff
@@ -298,7 +298,7 @@ The `scanf()` function will stop accepting input after the character `y` and sto
 <tr><td colspan="31" align="center">char<br/>name</td></tr>
 <tr><td>0</td>
     <td>1</td>
-    <td><strong>2</strong><td>
+    <td><strong>2</strong></td>
```
Comment (humphd) [inline docs/F-Refinements/character-strings.md:301]: Good catch.  Next time, do this in a separate PR, since this is unrelated to this fix.

File: docs/F-Refinements/two-dimensional-arrays.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ This chapter introduces two-dimensional arrays, describing their `syntax` and th
 
 A table is a useful analogy for describing a two-dimensional array. An entry in a table is identified by its row and column positions. Consider the row and column indices in the figure below. `The first index` refers to the `row` and `the second index` refers to the `column`.
 
-![5x5_2DArrayExample](https://cdn.discordapp.com/attachments/353432193730871296/911315058041171988/table_any.png)
+![5x5_2DArrayExample](img/2d array.png)
```
Comment (humphd) [inline docs/F-Refinements/two-dimensional-arrays.md:27]: `/img`

**Review by humphd — CHANGES_REQUESTED** [review]
Fantastic, thank you for jumping on this.  A couple small things.

File: docs/F-Refinements/two-dimensional-arrays.md (hunk 2 above)
Comment (Kevan-Y) [reply docs/F-Refinements/two-dimensional-arrays.md]: Fixed

File: docs/F-Refinements/character-strings.md (hunk 1 above)
Comment (Kevan-Y) [reply docs/F-Refinements/character-strings.md]: Will create an issue for those

**Comment by Kevan-Y** [comment]
Recent commit: I notice that there is an img folder that was created in the root folder. I moved those img to our static/img folder

**Review by humphd — APPROVED** [review]
I think this looks right, nice job.

**Review by CameronGray1210 — APPROVED** [review]
Nice!! Thank you for porting all those images.
